src/components/views/causes/services.ts

```
@@ -0,0 +1,53 @@
+// services/causesService.ts
+
+import { client } from '@/apollo/apolloClient';
+import { FETCH_ALL_CAUSES } from '@/apollo/gql/gqlCauses';
```

ESLint

[error] FETCH_ALL_CAUSES not found in '@/apollo/gql/gqlCauses'

(import/named)